cpp/include/Ice/Properties.h

```diff
@@ -192,6 +192,50 @@ namespace Ice
         std::vector<std::string> _optInPrefixes;
         mutable std::mutex _mutex;
     };
+
+    /// Creates an empty property set.
+    /// @return A new empty property set.
+    ICE_API PropertiesPtr createProperties();
```

Comment:
Moved from Initialize.h. Didn't change otherwise (except for the doc-comments).

Reply:
ins't this deprecated?